modules/editorial-metadata/editorial-metadata.php

```diff
@@ -46,10 +46,36 @@ public function init() {
 		// Register the taxonomy we use for Editorial Metadata with WordPress core
 		$this->register_taxonomy();
 
+		EditEditorialMetadata::init();
+
 		// Load CSS and JS resources that we probably need in the admin page
 		add_action( 'admin_enqueue_scripts', [ $this, 'action_admin_enqueue_scripts' ] );
 
-		EditEditorialMetadata::init();
+		// Load block editor JS
+		add_action( 'enqueue_block_editor_assets', [ $this, 'load_scripts_for_block_editor' ], 9 /* Load before custom status module */ );
```

Comment:
We *should* be able to use script dependencies rather than explicit priorities to be able to decide load order. In `load_scripts_for_block_editor`, maybe this would work:

```php
$asset_file = include VIP_WORKFLOW_ROOT . '/dist/modules/editorial-metadata/editorial-metadata-block.asset.php';
$dependencies = array_merge( $asset_file['dependencies'], [ 'vip-workflow-block-custom-status-script' ] );

wp_enqueue_script( 'vip-workflow-block-editorial-metadata-script', VIP_WORKFLOW_URL . 'dist/modules/editorial-metadata/editorial-metadata-block.js', $dependencies, $asset_file['version'], true );
```

I see this is also being done in the preview module (probably my code), and that could potentially be fixed as well.

Reply:
I'll give this a shot

Reply:
Based on my testing, this works! I'll make the change in the preview module as well

Reply:
This won't work for the preview module, it results in a blank block editor. I tried to dig into it and fix it to no avail so I've skipped making this change for the preview module. I'm guessing this is because the preview module depends on the custom status module so it gets into some kind of a weird loop. If EM ever depends on that, we might have the same issue